mindjunkies/forums/views.py

```
@@ -145,29 +149,91 @@ def get(self, request, *args, **kwargs):
         reply = get_object_or_404(Reply, id=reply_id)
         if not reply:
             reply = get_object_or_404(ForumComment, id=reply_id)
-        reply_form = ForumReplyForm()
+        replyForm = ForumReplyForm()
```

Ruff (0.8.2)

Variable `replyForm` in function should be lowercase

(N806)